File: Microsoft.CodeAnalysis.VisualBasic/Microsoft.CodeAnalysis.VisualBasic/LookupResultKind.cs
namespace Microsoft.CodeAnalysis.VisualBasic
{
	internal enum LookupResultKind
	{
		Empty,
		NotATypeOrNamespace,
		NotAnAttributeType,
		WrongArity,
		NotCreatable,
		Inaccessible,
		NotReferencable,
		NotAValue,
		NotAVariable,
		MustNotBeInstance,
		MustBeInstance,
		NotAnEvent,
		LateBound,
		EmptyAndStopLookup,
		WrongArityAndStopLookup,
		OverloadResolutionFailure,
		NotAWithEventsMember,
		Ambiguous,
		MemberGroup,
		Good
	}
}

File: Microsoft.CodeAnalysis.VisualBasic/Microsoft.CodeAnalysis/VisualBasic/LookupResultKind.vb
Imports System

Namespace Microsoft.CodeAnalysis.VisualBasic
	Friend Enum LookupResultKind
		Empty
		NotATypeOrNamespace
		NotAnAttributeType
		WrongArity
		NotCreatable
		Inaccessible
		NotReferencable
		NotAValue
		NotAVariable
		MustNotBeInstance
		MustBeInstance
		NotAnEvent
		LateBound
		EmptyAndStopLookup
		WrongArityAndStopLookup
		OverloadResolutionFailure
		NotAWithEventsMember
		Ambiguous
		MemberGroup
		Good
	End Enum
End Namespace